src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -5260,6 +5260,102 @@ def test_sql_managed_instance_mgmt(self, mi, rg):
         # test list sql managed_instance in the subscription should be at least 1
         self.cmd('sql mi list', checks=[JMESPathCheckGreaterThan('length(@)', 0)])
 
+class SqlManagedInstanceStartStopMgmtScenarioTest(ScenarioTest):
+    @AllowLargeResponse()
+    def test_sql_mi_startstop_mgmt(self):
+        rg = 'CustomerExperienceTeam_RG'
+        mi = 'clitestmilb5hsyvgolc22pa3zzf2urno3uwskko4us2mbcti2gebgawczstsm'
```

Comment:
Please update the test with `ResourceGroupPreparer` and `ManagedInstancePreparer` instead of using fixed test resources.

Reply:
We are having an issue with preparing a Managed Instance, since for this feature we require a Managed Instance on a v2 cluster, and our cluster tries to resize during every MI create, the creation lasts for >3hrs. Due to that, the CLI test that I initially wrote that prepares the MI gets an error about a refused connection: 
![image](https://github.com/Azure/azure-cli/assets/53649816/8d9ab288-785b-49af-a087-bd88d9a9d417)
I'm guessing it happens due to a timeout because creation takes too long. I will be addressing this in the next snap by preparing our test ring to not resize during create, which will shorten the duration of creation time.